R/modify_pd.R

```diff
@@ -0,0 +1,53 @@
+#' Update indention information of parse data
+#'
+#' @param pd A nested or flat parse table that is already enhanced with
+#'   line break and space information via [create_filler()] or
+#'   [create_filler_nested()].
+#' @param indent_by How many spaces should be added after the token of interest.
+#' @name update_indention
+NULL
+
+#' @rdname update_indention
+indent_round <- function(pd, indent_by) {
+  if (any(pd$token == "')'")) {
+    start <- 2
+    stop <- nrow(pd) - 1
+  } else {
+    start <- stop <- 0
+  }
+  pd$indent <- ifelse(seq_len(nrow(pd)) %in% start:stop, indent_by, 0)
```

Comment:
This might look clearer if we move this assignment to the `if` branch, and remove the `else` branch.